gap/oper.gi

```diff
@@ -1684,3 +1684,45 @@ function(digraph, list)
 
   return DigraphShortestDistance(digraph, list[1], list[2]);
 end);
+
+# 
+
+InstallMethod(DigraphClosure,
+"for a digraph and an integer",
+[IsDigraph, IsPosInt],
+function(digraph, k)
+  local out, adj_mat, degs, n, new_edge, row, i, j;
+
+  if not IsSymmetricDigraph(digraph) or DigraphHasLoops(digraph) then
+    ErrorNoReturn("Digraphs: DigraphClosure: usage,\n",
+                  "the graph must by symmetric and without loops,");
+  fi;
+
+  out := [];
+  adj_mat := BooleanAdjacencyMatrix(digraph);
+  for row in adj_mat do
+    Add(out, ShallowCopy(row));
+  od;
+  degs:= ShallowCopy(InDegrees(digraph));
+  n   := DigraphNrVertices(digraph);
+  while true do
```

Comment:
In my opinion, the previous line should say `new_edge := true` and this while loop should say `while new_edge do`. This removes the need for the `if` statement at the end of the this `while` loop.